web/packages/teleterm/src/ui/ConnectMyComputer/DocumentConnectMyComputerStatus/DocumentConnectMyComputerStatus.story.tsx

```diff
@@ -102,6 +102,12 @@ const doc: types.DocumentConnectMyComputerStatus = {
   rootClusterUri: cluster.uri,
   kind: 'doc.connect_my_computer_status',
 };
+const workspace: types.Workspace = {
```

Comment:
If it needs to be defined at the top level because of issues with reloading, wouldn't it make sense to put it inside the component that needs it? That should be okay for story components.

Or maybe we could at least put a comment as to why it needs to be defined outside of the component.

Reply:
>If it needs to be defined at the top level because of issues with reloading, wouldn't it make sense to put it inside the component that needs it?

But we have 6 story components, so it would add a bit of boilerplate. I think we can solve this by wrapping `workspace` in `useMemo` with a comment explaining why.